tests/integration/test_charm.py

```diff
@@ -58,8 +62,30 @@ async def test_with_ingress(
     traefik_ip = next(await get_unit_ips(traefik_app_name))
     response = requests.get(
         f"http://{traefik_ip}",
-        headers={"Host": f"{ops_test.model_name}-{app.name}.{external_hostname}"},
+        headers={"Host": f"{ops_test.model_name}-{gh_app.name}.{external_hostname}"},
         timeout=5,
     )
     assert response.status_code == 200
     assert "GitHub Actions Exporter" in response.text
+
+
+async def test_with_nginx_route(
+    ops_test: OpsTest,
+    gh_app: Application,  # pylint: disable=unused-argument
+    nginx_integrator_app,  # pylint: disable=unused-argument
```

Comment:
Would `pytest.mark.usefixtures` work here? https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#use-fixtures-in-classes-and-modules-with-usefixtures

Reply:
Sure, changed.